health-reform-observer.csl

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Health Reform Observer</title>
+    <id>http://www.zotero.org/styles/health-reform-observer</id>
+    <link rel="self" href="http://www.zotero.org/styles/health-reform-observer"/>
+    <link rel="documentation" href="https://escarpmentpress.org/hro-ors/pages/view/how-to-format-ms"/>
+    <link href="http://www.zotero.org/styles/elsevier-harvard" rel="template"/>
+    <contributor> 
+   <name>Naomi Scott</name>
+   <email>scottnb@mcmaster.ca</email>
+   </contributor>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
```

Comment:
@scottnb, again please add `<category field="medicine"/>` (and keep `<category field="social_science"/>`)